lib/web_ui/dev/edge_installation.dart

```diff
@@ -68,12 +71,73 @@ Future<BrowserInstallation> getEdgeInstallation(
     // Since Edge is included in Windows, always assume there will be one on the
     // system.
     infoLog.writeln('Using the system version that is already installed.');
+    final EdgeLauncher edgeLauncher = EdgeLauncher();
+    if (edgeLauncher.isInstalled) {
+      infoLog.writeln('Launcher installation was skipped, already installed.');
+    } else {
+      infoLog.writeln('Installing MicrosoftEdgeLauncher');
+      await edgeLauncher.install();
+      infoLog.writeln(
+          'Installations complete. To launch it run ${edgeLauncher.executable}');
+    }
+
     return BrowserInstallation(
       version: 'system',
-      executable: PlatformBinding.instance.getCommandToRunEdge(),
+      executable: io.Directory(path.join(
+              edgeLauncher.launcherInstallationDir.path,
+              '${PlatformBinding.instance.getCommandToRunEdge()}'))
+          .path,
     );
   } else {
     infoLog.writeln('Unsupported version $requestedVersion.');
     throw UnimplementedError();
   }
 }
+
+class EdgeLauncher {
+  /// Url for downloading  `MicrosoftEdgeLauncher`.
+  ///
+  /// Only useful in Windows, hence not added to [PlatformBinding].
+  final String windowsEdgeLauncherDownloadUrl =
+      'https://github.com/MicrosoftEdge/edge-launcher/releases/download/1.2.0.0/MicrosoftEdgeLauncher.exe';
+
+  /// Path to the directory contains `MicrosoftEdgeLauncher.exe`.
+  io.Directory get launcherInstallationDir => io.Directory(
+        path.join(environment.webUiDartToolDir.path, 'microsoftedgelauncher'),
```

Comment:
I think the path should contain the version number. Otherwise, installation will be skipped even after we upgrade to a new version.

Reply:
done.